playbooks/provisioning/openstack/post-provision-openstack.yml

```diff
@@ -90,3 +99,18 @@
         line: "IP4_NAMESERVERS={{ hostvars['localhost'].private_dns_server }}"
   roles:
     - node-network-manager
+
+- name: Show configured DNS nsupdate keys for future use as an external one
```

Comment:
The end of the post-provision playbook is the current separator between the provision and deployment, it should be no lost in the noise hopefully. I'll explore the option with a separate role in infra-ansible tho. Thanks!